test/events.test.tsx

```diff
@@ -118,11 +118,70 @@ describe('Events', () => {
     await userEvent.pointer({ target: getCellsAtRowIndex(0)[1], keys: '[MouseRight]' });
     expect(getCellsAtRowIndex(0)[1]).toHaveAttribute('aria-selected', 'true');
   });
+
+  it('should call onSelectedCellChange when cell is clicked', async () => {
+    let idx: number;
+    let rowIdx: number;
+    let row: Row;
+    let column: CalculatedColumn<Row> | undefined;
+
+    render(
+      <EventTest
+        onSelectedCellChange={(args) => {
+          idx = args.idx;
+          rowIdx = args.rowIdx;
+          row = args.row;
+          column = args.column;
+        }}
+      />
+    );
+
+    const expectOnCellSelectedFired = () => {
+      expect(getCellsAtRowIndex(0)[1]).toHaveAttribute('aria-selected', 'true');
+      expect(idx).toBe(1);
+      expect(rowIdx).toBe(0);
+      expect(row).toStrictEqual({
+        col1: 1,
+        col2: 'a1'
+      });
+      expect(column?.key).toBe('col2');
+    };
+
+    // Selected by click
+    await userEvent.click(getCellsAtRowIndex(0)[1]);
+    expectOnCellSelectedFired();
+
+    // Selected by double click
+    await userEvent.dblClick(getCellsAtRowIndex(0)[1]);
```

Comment:
```suggestion
    await userEvent.dblClick(getCellsAtRowIndex(0)[0]);
```

Let's change the selected cell for each action.

Reply:
To combine the test, the selected cell has to be the same through all actions to select (click, double click, right-click, arrow key, and tab key). To select a cell by tab key, the selected cell cannot be (0, 0) because you can only move to right cell by tab key. The column has to be 1.

We can add arguments to `expectOnCellSelectedFired()` to change selected cells by each actions. However, according to some experts, I hear you should make the test code less DRY because making the test code too complex causes bugs in test code. Therefore, we should avoid adding the arguments to `expectOnCellSelectedFired()`, in my opinion.

However, I don't have strong opinion on this, so let me know if you prefer to add arguments to `expectOnCellSelectedFired()` so that we can select different cells by the actions.